static/js/components/ProgressWidget.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+export default class ProgressWidget extends React.Component {
+
+  props: {
+    actual: number,
+    total:  number
+  };
+
+  // this is temporary, we can remove it once it is integrate into learners dashboard
+  static defaultProps = {
+    actual: 3,
+    total:  5
+  };
+
+  render() {
+    const { actual, total } = this.props;
+
+    return (
+      <div className="progress-widget">
+        <div className="row-progress-widget">
+          <p className="text heading-text">Progress</p>
```

Comment:
this `<p>` tag is unnecessary, you can just put `text heading-text` on the wrapping div.